lib/settings.js

```diff
@@ -9,7 +9,7 @@ const rootDir = process.cwd();
 const sourcePath = './assets/Zed/';
 const publicPath = '/assets/';
 const sourceDir = path.resolve(sourcePath);
-const publicDir = path.resolve(path.join('public/Zed', publicPath));
+const publicDir = path.resolve(path.join('public/Backoffice', publicPath));
```

Comment:
Is this switching from Zed to Backoffice here?
What about customers that are using still the old path?

Reply:
Seems like a BC break to me.
@tolerants are you planning to release a major for this library?

Reply:
@tolerants can you please clarify a BC way to have both Zed and Backoffice working?
My guess would be to detect this somehow from the current project setup or alike?

Reply:
Since this oryx-for-zed is not a direct dependency in composer.json of project, I doubt it. and then even a major wouldnt work. It would be silently pulled also for the "old way".

Best to make sure we copy it "twice" now for both.

Reply:
So it's a minor.

Reply:
Aha, I see. OK, I trust your assessment here.